Add link to Kitely update giver

codeviolet opened the pull request:
Unfortunately there doesn't seem to be a straight-forward and reliable way to completely automate redelivery in Hypergrid for users who choose to purchase through Kitely Market. I have however placed an update giver in the 'AVsitter sandbox' region of Kitely where avatars can get the latest packaged OpenSim version. Whenever an avatar receives the package from Kitely market their details are manually added to the update giver. After this, they've been added, they can return at any time to click the updater to get the latest package.

codeviolet committed:
Link to Kitely update giver

Sei-Lisa commented:
I just saw your edit. Does access to the sim still require the group? I can't log in and check.

On a side note, GitHub doesn't notify me about edits (there was no such option last time I checked). If it's a substantial one that you expect to be seen, please consider adding it as a new comment, or adding a new comment notifying about the existence of edits.

codeviolet commented:
I've just taken another look at the arrangements for the sim and changed them somewhat to the following:

* Small parcel in the center of the sim owned by AVsitter resident where only that avatar should be able to rez. In this small parcel is placed the update giver and a basic llDialog greeter.

* Remaining (large) parcel owned by the group *AVsitter sandbox*. Access to everyone, build everyone. 120 min autoreturn.

This means that access to the *AVsitter sandbox* **group** should be restricted only to those people who have legitimate need to bypass the autoreturn and rez permanent structures or make use of group roles. General users do not need to be group members to use the sandbox. Consequently I have removed that group from search and "anyone can join".

I think I prefer this setup as opposed to requiring group membership, but let me know if you think it should change. As group owner feel free to tweak the group or sandbox parcel / settings any way you like too.

If we need a separate Kitely group for communication with users then that would need to be a different group under this setup (however I doubt that will be needed).

Let me know if I have overlooked anything with this setup...

Thanks for the confirmation that edits don't get notifications. It was not needing your urgent attention and I was still working it through in my head so I just added my note to the original, empty, comment.

The only thing it may help for you to be aware of is that the update giver in Kitely is manually updated by me whenever someone decides to get AVsitter from Kitely Market (Kitely don't offer an ANS system or buyer-initiated redelivery as yet). In case someone complains that it's not working then it may just be that I have yet to add their avatar name to the list :)
